_scripts/l10n/prime.sh

```diff
@@ -12,16 +12,16 @@ if [ -z "$PACKAGE" ]; then
 fi
 
 # Move to monorepo root
-cd "$(dirname "$0")/../.."
-ROOT_FOLDER=$(pwd)
+ROOT_FOLDER=$(git rev-parse --show-toplevel)
```

Comment:
A little concerned about this. WFM on *nix, but haven't tried on windows. Anyone happen to have ideas for a better approach?

Reply:
Ah hmm, looks like WSL on Windows isn't a fan...

https://stackoverflow.com/questions/76838262/git-rev-parse-show-toplevel-not-working-for-worktree-within-wsl

Reply:
I have no windows computer in my house, but otherwise this PR looks good